catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/types/CoreAttributes.java

```diff
@@ -84,7 +84,7 @@
         descriptors.add(new AttributeDescriptorImpl(METADATA,
                 true /* indexed */,
                 true /* stored */,
-                true /* tokenized */,
+                false /* tokenized */,
```

Comment:
I don't agree with this change.  Metadata should and is currently tokenized.  Searches would have to match the entire XML blob to match if tokenization is false.